modal/_container_entrypoint.py

```diff
@@ -36,7 +37,7 @@
 from ._blob_utils import MAX_OBJECT_SIZE_BYTES, blob_download, blob_upload
 from ._function_utils import LocalFunctionError, is_async as get_is_async, is_global_function
 from ._proxy_tunnel import proxy_tunnel
-from ._pty import exec_cmd, run_in_pty
+from ._pty import exec_cmd
```

Comment:
do we still need `exec_cmd`?

Reply:
hmm good point, I don't think so!

Reply:
it's used on L681:
```python
    if pty_info.pty_type == api_pb2.PTYInfo.PTY_TYPE_SHELL:
        fun = exec_cmd
```

but I think that branch might be dead now? but not sure